src/main/resources/hudson/plugins/jira/JiraEnvironmentVariableBuilder/config.jelly

```diff
@@ -1,5 +1,10 @@
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
     <j:if test="${descriptor.hasIssueSelectors()}">
         <f:dropdownDescriptorSelector field="issueSelector" title="${%Issue selector}"/>
-    </j:if>  
+    </j:if>
+    <f:advanced>
+        <f:entry title="${%Issues size variable name}" field="issuesSizeVariableName">
+            <f:textbox default="JIRA_ISSUES_SIZE"/>
```

Comment:
better to use word COUNT instead of SIZE.
size is usually used when creating an object of specific size to show how many elements can fit in collection, not how many there are